content/blog/monthly-july-2020.md

```diff
@@ -0,0 +1,80 @@
+---
+title: This Month in TiKV - July 2020
+date: 2020-07-30
+author: TiKV Authors
+---
+The warmest month in the Northern Hemisphere comes to an end soon. It’s time to wrap up everything we have achieved in this month!
+
+Let’s get started.
+
+## News
+
+TiKV is up for graduation from CNCF! This is a milestone that would not be possible without you! If you would like to support TiKV in the vote, follow the [instructions](https://lists.cncf.io/g/cncf-toc/join) to join TOC and post your replies [here](https://lists.cncf.io/g/cncf-toc/message/4902)!
+
+[@breeswish](https://github.com/breeswish) has talked about TiKV’s development path and new features in the 4.0 GA version at Kubernetes Paris meetup. In case you missed it, the recorded video is available [here](https://www.youtube.com/watch?v=pAattr8cwSY&list=PLfR-valDYipfVN7ICQF_DnlZHcTY7O1F3&index=10&t=0s).
```

Comment:
```suggestion
[@breeswish](https://github.com/breeswish) talked about TiKV’s development path and new features in the 4.0 GA at Kubernetes Paris meetup. In case you missed it, the recorded video is available [here](https://www.youtube.com/watch?v=pAattr8cwSY&list=PLfR-valDYipfVN7ICQF_DnlZHcTY7O1F3&index=10&t=0s).
```